ReactAndroid/src/main/java/com/facebook/react/ReactActivity.java

```diff
@@ -0,0 +1,212 @@
+package com.facebook.react;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.os.Build;
+import android.os.Bundle;
+import android.os.Handler;
+import android.provider.Settings;
+import android.view.KeyEvent;
+import android.widget.EditText;
+import android.widget.Toast;
+
+import com.facebook.common.logging.FLog;
+import com.facebook.react.common.ReactConstants;
+import com.facebook.react.modules.core.DefaultHardwareBackBtnHandler;
+import com.facebook.react.shell.MainReactPackage;
+
+import java.util.List;
+
+import javax.annotation.Nullable;
+
+/**
+ * Base Activity for React Native applications.
+ */
+public abstract class ReactActivity extends Activity implements DefaultHardwareBackBtnHandler {
+
+  private static final String REDBOX_PERMISSION_MESSAGE =
+      "Overlay permissions needs to be granted in order for react native apps to run in dev mode";
+
+  private @Nullable ReactInstanceManager mReactInstanceManager;
+  private LifecycleState mLifecycleState = LifecycleState.BEFORE_RESUME;
+  private boolean mDoRefresh = false;
+
+  /**
+   * @return the name of the bundle in assets. If this is null, the app will only work with dev
+   * support on and will always try to load the JS bundle from the packager server.
```

Comment:
@mkonicek Yeah, I think that's what it means, as this is the file name of the bundle and it can't be loaded if this is not specified.